django_imgix/templatetags/imgix_tags.py

```diff
@@ -7,7 +7,7 @@
     # Python 3 location
     from urllib.parse import urlparse
 
-from django.utils import six
```

Comment:
Hey 👋 @phil-bell, thanks for submitting this PR! Overall it looks great. We'd actually like to drop `six` altogether as we drop support for python-2.7 (and really anything below python-3.6 as those versions have all reached EOL).

Would you mind making the following changes:

- remove the import for `six` and the requirement in `setup.py`
- remove calls to `six.b(...)` and `six.string_types`
  - `six.b(...)` can be replaced with `b'...'`
  - `isinstance(size, six.string_types)` can be replaced with `isinstance(size, str)`
  - instances of `re.compile(six.b(r'...'))` can be replaced with `re.compile(r'...')`
- and finally, roll the patch bump back to `'1.2.0'` (we usually do this in a second subsequent step)

After that we'll merge this in. CI won't pass, but as long as the py37 portions pass we're good to go. Dropping py27 support is a bit out of scope for this PR, so we'll take care of that after this gets merged.

Again, thanks!!!

Reply:
Hi @ericdeansanchez 

Thanks for the guidance, I removed `six`, rolled the imgex version back to `1.2.0 fixed fixed the tests for py3.7.